Example/EmissionTests/TestApps.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+'use strict';
+
+import Emission from 'emission';
+import React from 'react-native';
+const { TestModule } = React.NativeModules;
+
+class OpaqueImageViewTest extends React.Component {
+  constructor() {
+    super();
+    this.createSnapshot = this.createSnapshot.bind(this);
+  }
+
+  createSnapshot() {
```

Comment:
Can there be one createSnapshot() call per file? And does that mean only one snapshot per file?